.circleci/config.yml

```diff
@@ -57,6 +57,12 @@ jobs:
       - run: bundle exec rake db:create
       - run: bundle exec rake db:schema:load
 
+      - run:
+          name: run diff on schema.json
+          command: |
+            bundle exec rake graphql:schema:dump
+            git diff --quiet schema.graphql
+
```

Comment:
be careful with rebasing, looks like this got pulled into your branch by mistake.  Do you know what happened here?

Reply:
Hm, I was trying to fix my branch which diverged into two separate ones. My terminal stated it had '2 and 17 different commits each'. This is probably what was causing some errors while rebasing.